### ISSUE TITLE
SIGCHLD not accepted by skills core

### ISSUE THREAD

**Issue by SoloVeniaASaludar** [MycroftAI/mycroft-core#571]
When a child launched with "subprocess.Popen" finishes, it jumps to state "defunct" until parent (Mycroft core) accepts the signal "SIGCHLD". Example:

```
$ python
>>> import subprocess
>>> x=subprocess.Popen(["sleep","1"])
```

(in another terminal)
```
$ ps -ef | egrep [s]leep
svas      17160 17143  0 16:17 pts/7    00:00:00 [sleep] <defunct>
```

if the signal is accepted and ignored, the process finishes correctly:

```
$ python
>>> import subprocess
>>> import signal
>>> signal.signal(signal.SIGCHLD,signal.SIG_IGN)
0
>>> x=subprocess.Popen(["sleep","1"])
```
```

$ ps -ef | egrep [s]leep
```

For this reason it is important that core.py in skills handles this signal.

**Comment by penrods** [comment]
@SoloVeniaASaludar Fix for this was merged into build 0.8.8.  Unless you see the problem still, I'll close this issue.

**Comment by SoloVeniaASaludar** [comment]
Thanks a lot.
Issue closed.
